src/content/en/updates/2018/09/reportingapi.md

```diff
@@ -0,0 +1,521 @@
+  response.set('Report-To', `{
+    "max_age": 2592000,
+    "endpoints": [{
+      "url": "https://reporting-observer-api-demo.glitch.me/reports"
+    }],
+  }, {
+    "group": "csp-endpoint",
+    "max_age": 2592000,
+    "endpoints": [{
+      "url": "https://reporting-observer-api-demo.glitch.me/csp-reports"
+    }],
+  }, {
+    "group": "network-errors",
+    "max_age": 2592000,
+    "endpoints": [{
+      "url": "https://reporting-observer-api-demo.glitch.me/network-reports"
+    }]
+  }`);
+
+  response.sendFile('./index.html');
+});
+
+function echoReports(request, response) {
+  // Record report in server logs or otherwise process results.
+  for (const report of request.body) {
+    console.log(report.body);
+  }
+  response.send(request.body);
+}
+
+app.post('/csp-reports', (request, response) => {
+  console.log(`${request.body.length} CSP violation reports:`);
+  echoReports(request, response);
+});
+
+app.post('/network-reports', (request, response) => {
+  console.log(`${request.body.length} Network error reports:`);
+  echoReports(request, response);
+});
+
+app.post('/reports', (request, response) => {
+  console.log(`${request.body.length} deprecation/intervention reports:`);
+  echoReports(request, response);
+});
+
+
+const listener = app.listen(process.env.PORT, () => {
+  console.log(`Your app is listening on port ${listener.address().port}`);
+});
+```
+
+## What about ReportingObserver? {: #ro }
+
+Although both are part of the same [Reporting API spec][spec],
+[ReportingObserver][reportingobserver] and the `Report-To` header have overlap
+with each other but enable slightly different uses cases.
+
+`ReportingObserver` is a JavaScript API that can observe simple
+client-side warnings like deprecation and intervention. Reports
+are not automatically sent to a server (unless you choose to do so in the callback):
+
+```js
+const observer = new ReportingObserver((reports, observer) => {
+  for (const report of reports) {
+    // Send report somewhere.
+  }
+}, {buffered: true});
+
+observer.observe();
+```
+
+More sensitive types of errors like CSP violations and network errors
+cannot be observed by a `ReportingObserver`. Enter `Report-To`.
+
+The `Report-To` header is more powerful in that it can capture
+[more types](#reportypes) of error reports (network, CSP, browser crashes)
+in addition to the ones supported in `ReportingObserver`. Use it when you
+want to automatically report errors to a server or capture errors
+that are otherwise impossible to see in JavaScript (network errors).
+
+## Conclusion
+
+Although the Reporting API is a ways out from shipping in all browsers, it's
+a promising tool for diagnosing issues across your site.
+
+Warnings that get logged to the DevTools console are super helpful
+but have limited value to you as the site author. That's because they're
+local to the user's browser! The Reporting API changes this. Use it to
+configure, detect, and report to a server even errors even when your own
```

Comment:
Use it to configure, detect, and report to a server errors even when your own code cannot.